### DISCUSSION TITLE
Custom Auth Package not working without keystone/auth package

### DISCUSSION THREAD

**Discussion by muhad-bk** [keystonejs/keystone#7751] [Questions] [answered]
### I created a custom auth module for keystone (k6-auth-otp)

[k6-auth-otp](https://www.npmjs.com/package/k6-auth-otp)

And it shows the error like this 

![image](https://user-images.githubusercontent.com/75353837/179305394-7de37848-4f2a-4e10-b263-f34132a6a7bc.png)

**Comment by borisno2** [comment, marked as answer]
Hi @muhad-bk 
I assume you copied the `keystone-6/auth` package and are making some changes? From the error, it looks like the `signin` and `init` templates that get copied for use in the admin UI are still referencing the `keystone-6/auth` package - https://github.com/keystonejs/keystone/blob/main/packages/auth/src/templates/signin.ts#L13

**Reply by muhad-bk** [reply]
Thanks @borisno2
